feebleline.el

```diff
@@ -92,6 +93,7 @@
 (defvar feebleline--home-dir nil)
 (defvar feebleline--msg-timer)
 (defvar feebleline--mode-line-format-previous)
+(defvar feebleline-last-error-shown nil)
```

Comment:
Internal variables usually have two dashes in elisp. This is the only remark I have.